hak33m16/ladder
# Adds placeholder sfx and mute/unmute button

To test:
- Once game is started, sound should be muted by default and sound toggle button present in bottom right
- Sound button should toggle between mute / unmute icons when clicked
- Sound button should toggle between muting / unmuting sounds
- A sound will play when character jumps
- A sound will play when character collects a coin

I also attempted to add start menu music, but Chrome will not allow sounds to autoplay without human interaction first, which is why I decided to originally add the sound toggle button.  I have not decided the best place to insert the code that controls the sound button rendering and mute/unmute functionality because I need both access to the canvas and the ability to hold the global mute state. The Howler.js library has a function for setting the mute state, but doesn't seem to expose a getter that returns the current mute state.

I think this is a good stopping point, but would like to improve the sound system further in a future PR.
